tests/User/UserTest.php

```diff
@@ -289,15 +291,17 @@ public function testUpdateQuotaIndividualProvided() {
 
 		$connection->expects($this->at(0))
 			->method('__get')
-			->with($this->equalTo('ldapQuotaDefault'))
-			->will($this->returnValue(''));
+			->with($this->equalTo('ldapQuotaAttribute'))
+			->will($this->returnValue('myquota'));
 
+		/* Having a quota set this won't be used
 		$connection->expects($this->at(1))
 			->method('__get')
-			->with($this->equalTo('ldapQuotaAttribute'))
-			->will($this->returnValue('myquota'));
+			->with($this->equalTo('ldapQuotaDefault'))
+			->will($this->returnValue(''));
```

Comment:
It's commented on purpose, to make clear that the LDAP default quota won't be fetched in that case.
It doesn't seem possible to check if the method won't be called with the specified parameters, so I'm not sure if leave this commented or remove it.